apps/verification-portal/src/lib/components/SocialShare.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
   import HexagonIcon from './icons/HexagonIcon.svelte';
   import LL from '$lib/shared/i18n/i18n-svelte';
+  import { shareFile } from '$lib/features/actions';
```

Comment:
https://svelte.dev/docs/svelte-action is a reserved interface, so probably just a sharing feature.